lib/gems/pending/appliance_console/cli.rb

```diff
@@ -145,6 +151,7 @@ def run
       install_ipa if install_ipa?
       install_certs if certs?
       extauth_opts if extauth_opts?
+      server_mode if server_state?
```

Comment:
I think these methods would be better if they were named as verbs.

```ruby
set_server_state if set_server_state?
```

Reads a lot better to me.